content/2022-09-19-order-flow-auctions-and-centralisation-2.mdx

```diff
@@ -2,7 +2,7 @@
 slug: order-flow-auctions-and-centralisation-II
 title: order flow, auctions and centralisation II - order flow auctions
 authors: [quintus]
-tags: [flashbots, pbs, order-flow]
+tags: [pbs, order-flow, ofa, eof, fee escalator]
```

Comment:
i think we shouldn't introduce new tags too easily, but reconcile all posts into 5-8 tags? it's somewhat strange to click on a tag and only have a single post showing up